### DISCUSSION TITLE
(已解决)添加pikpak出现invalid_request错误

### DISCUSSION THREAD

**Discussion by satellas** [AlistGo/alist#1226] [Q&A]
如图，添加账号时只填写了用户名和密码，打开文件夹出现错误
![Snipaste_2022-06-13_21-36-31](https://user-images.githubusercontent.com/78683129/173367243-40f4d159-2700-4a4e-921d-1914932a09e7.png)

**Comment by Jue-Xu** [comment]
挂梯子了吗

**Reply by satellas** [reply]
挂了

**Reply by Mosazh** [reply]
使用邮箱登录试试

**Reply by satellas** [reply]
>感谢老哥，现在可以了
